mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/annotations/Query.java

```diff
@@ -141,4 +141,6 @@
    * was built with. See the top-level javadocs of this class for more explanations.
    */
   String value();
+
+  // todo do we need NullSavingStrategy to Query? (I think that it only retrieves data)
```

Comment:
I am not sure why in JIRA ticket there was info that we need to handle `NullSavingStrategy` for `@Query`.
In `QueryIT` I see that all queries are `SELECT` or `DELETE` so `nullSavingStrategy` has no influence over them.
Do we plan to support `INSERT ..` in `@Query` annotation?

Reply:
`@Query` is completely free form, it can be whatever you put in the query string.